aspnet/conceptual-overview/aspnet.rst

```diff
@@ -104,8 +124,23 @@ ASP.NET Core uses a new configuration model for handling of simple name-value pa
 
 See :doc:`/fundamentals/configuration` for more details on the new configuration system and :doc:`/fundamentals/environments` for details on how to work with environments in ASP.NET Core.
 
+Build web UI and web APIs using MVC
```

Comment:
@shaunluttin lc web & internet has been our standard for some time. I need to put it in the style guide.